app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationScreen.kt

```diff
@@ -27,12 +27,13 @@ import com.wire.android.ui.home.conversations.model.AttachmentBundle
 import com.wire.android.ui.home.conversations.model.MessageViewWrapper
 import com.wire.android.ui.home.messagecomposer.MessageComposeInputState
 import com.wire.android.ui.home.messagecomposer.MessageComposer
+import com.wire.android.util.dialogErrorStrings
+import com.wire.android.util.permission.rememberCallingRecordAudioRequestFlow
 import kotlinx.coroutines.launch
 
 @Composable
-fun ConversationScreen(
-    conversationViewModel: ConversationViewModel
-) {
+fun ConversationScreen(conversationViewModel: ConversationViewModel) {
+    val audioPermissionCheck = AudioPermissionCheckFlow(conversationViewModel)
```

Comment:
in my opinion it will be more readable if we directly do ti like this here : 

```
  rememberCallingRecordAudioRequestFlow(onAudioPermissionGranted = {
        conversationViewModel.navigateToInitiatingCallScreen()
    }) {
        //TODO display an error dialog
    }
  ```

Reply:
I don't want to make OngoingCallContent as a God method, let's keep it here 🙂